<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workWidget</class>
 <widget class="QWidget" name="workWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="workLayout">
   <item>
    <widget class="QPlainTextEdit" name="workMessages">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="documentTitle">
      <string notr="true">Progress</string>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="workProgress">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
